learn2learn/utils/lightning.py

```diff
@@ -3,17 +3,139 @@
 """
 Some utilities to interface with PyTorch Lightning.
 """
-
+from typing import Optional, Callable
+import learn2learn as l2l
 import pytorch_lightning as pl
+from torch.utils.data._utils.worker import get_worker_info
+from torch.utils.data import IterableDataset
 import sys
 import tqdm
 
+class TaskDataParallel(IterableDataset):
 
-class EpisodicBatcher(pl.LightningDataModule):
+    def __init__(
+        self,
+        tasks: l2l.data.TaskDataset,
+        epoch_length: int,
+        devices: int = 1,
+        collate_fn: Optional[Callable] = None
+    ):
+        """
+        This class is used to sample epoch_length tasks to represent an epoch.
+
+        It should be used when using DataParallel
+
+        Args:
+            taskset: Dataset used to sample task.
+            epoch_length: The expected epoch length. This requires to be divisible by (num_workers * world_size).
```

Comment:
Second sentence of the description is not for `TaskDataParallel`.

Reply:
Done !